assets/css/style-freenet.css

```diff
@@ -115,6 +115,8 @@ border-radius: 0px;
 transition: all 0.5s ease-in-out;
 margin: 25px 5px 5px 0px;
 margin: 5px;
+position: relative;
+z-index: 1100;
```

Comment:
This seems to apply to all buttons, not just the download button. You might want to put these rules into a selector by id, or introduce another CSS class to cover this functionality if you feel this is still required.